edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/DeviceScopeIdentitiesCache.cs

```diff
@@ -51,6 +51,8 @@ public sealed class DeviceScopeIdentitiesCache : IDeviceScopeIdentitiesCache
 
         public event EventHandler<ServiceIdentity> ServiceIdentityUpdated;
 
+        public event EventHandler<IList<string>> ServiceIdentitiesUpdated;
```

Comment:
maybe augment one of the existing tests for this class to check this callback?  I think there's at least one test that validates the other two events already.